corehq/apps/enterprise/templates/enterprise/project_dashboard.html

```diff
@@ -1,25 +1,23 @@
 {% extends "hqwebapp/bootstrap5/base_section.html" %}
 {% load hq_shared_tags %}
 {% load i18n %}
-
-{% block page_title %}{{ current_page.title }}{% endblock %}
-
```

Comment:
I don't think I'm in favor of this change. While somewhat redundant, we are used to having pages with some kind of introductory paragraph, and that would now be missing here. Also, as I mentioned, I think ops relies on this title to distinguish enterprise console pages from each other. It's a nice additional check to see that you're looking at the enterprise console for specifically X, when you might have forgotten that you meant to be looking at Y.

Lastly, I also just prefer the look of the top line of text.

Reply:
Reverted!